config/mock_responses/adoption.json

```diff
@@ -0,0 +1,39 @@
+{
+  "results": [
+    {
+      "technology_name": "Wordpress",
+      "origins": [
```

Comment:
@tunetheweb is it correct that we'll have one api per metric? or will adoption, metadata, CWVs, lighthouse audits, etc. all come as a result from the same call?

Reply:
As per above, I've not started playing with it, but I think we should have one API call per technology. And include dates, mobile, desktop, Core Web Vitals, Lighthouse...etc. in it.

`all` will also count as a technology too.

Reply:
As I'm configuring some of the components now, it feels like it would indeed be easier to have one API call per tech and then include everything in there. 

Would also be useful if the results will be formatted more or less the same way, so we can do something reusable along the lines of `tech.results[metric][selected_client]` (with metric being eg. _origins_, _good_cwvs_, _lighthouse_performance_, etc) to render the values.